templates/home/api-fr.html

```diff
@@ -0,0 +1,214 @@
+</details>
+
+<h3 id="promotional-banner">Promotional banner</h3>
+<p>The promotional banner are over, at the bottom, of the home page banner background image.</p>
+<dl>
+	<dt>Version</dt>
+	<dd>2</dd>
+	<dt>Updated</dt>
+	<dd>2019-05-09</dd>
+	<dt>Notes</dt>
+	<dd>Provisional color was removed.</dd>
+</dl>
+<pre><code>&lt;div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
+	&lt;div class="container p-0 p-sm-3"&gt;
+		&lt;div class="well header-rwd brdr-0 brdr-rds-0 text-white bg-gctheme"&gt;
+			&lt;h1 property="name" id="wb-cont"&gt;Canada.ca&lt;/h1&gt;
+			&lt;p class="mrgn-tp-md"&gt;The official website of the Government of Canada&lt;/p&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+	&lt;div class="bg-gctheme opct-90"&gt;
+		&lt;div class="container"&gt;
+			&lt;a class="figcaption text-white" href="#"&gt;
+				&lt;figure class="clearfix"&gt;
+					&lt;div&gt;&lt;p class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;Featured:&lt;/p&gt;&lt;/div&gt;
+					&lt;figcaption class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;[Promotion title]&lt;/figcaption&gt;
+				&lt;/figure&gt;
+			&lt;/a&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/div&gt;
+</code></pre>
+<h4>Previous version</h4>
+<details>
+	<summary>Version 1 - 2018-12-27</summary>
+	<pre><code>&lt;div class="home-banner"&gt;
+		&lt;div class="container"&gt;
+			&lt;div class="innerbox"&gt;
+				&lt;h1 property="name" id="wb-cont"&gt;Canada.ca&lt;/h1&gt;
+				&lt;p&gt;The official website of the Government of Canada&lt;/p&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+		&lt;div class="promo"&gt;
+			&lt;div class="container"&gt;
+				&lt;a class="figcaption" href="#"&gt;
+					&lt;figure class="clearfix"&gt;
+						&lt;div&gt;&lt;p class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;Featured:&lt;/p&gt;&lt;/div&gt;
+						&lt;figcaption class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;[Promotion title]&lt;/figcaption&gt;
+					&lt;/figure&gt;
+				&lt;/a&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+	</code></pre>
+	<h5>Provisional</h5>
+	<p>Support for a set of predefined color to match the season. The intend is to set the color of the inner box with the site title to match the promotional banner background color.</p>
+	<p>The promotional color theme are set on the <code>div.home-banner</code>. Here the supported theme with their corresponding CSS class name</p>
+	<ul>
+		<li>(default) Sky - <code>sky</code></li>
+		<li>Earth - <code>earth</code></li>
+		<li>Vegetation - <code>vegetation</code></li>
+		<li>Night - <code>night</code></li>
+		<li>Forest - <code>forest</code></li>
+		<li>Autumn - <code>autumn</code></li>
+	</ul>
+</details>
+
+<h3 id="your-government">Your government</h3>
+<dl>
+	<dt>Version</dt>
+	<dd>2.0</dd>
+	<dt>Updated</dt>
+	<dd>2022-04-27</dd>
+	<dt>Notes</dt>
+	<dd>Links description was removed.</dd>
+</dl>
+<p>There are 8 links in this section presented in 2 columns.</p>
+<pre><code>&lt;section class="home-your-gov well well-sm brdr-0"&gt;
```

Comment:
The well-bold seems missing
```suggestion
<pre><code>&lt;section class="home-your-gov well well-bold well-sm brdr-0"&gt;
```